### ISSUE TITLE
Error when Searching in Japan, Works normal in US

### ISSUE THREAD

**Issue by Forza4Development** [AHAAAAAAA/PokemonGo-Map#1096]
## Expected Behavior
Should be scanning

## Current Behavior
Scans the first step, then throws an error.

## Possible Solution
JP Locale?

## Steps to Reproduce (for bugs)
Search using a Japanese Address

C:\Users\XXXX\Documents\PokemonGoMap\PokemonGo-Map-develop>C:\Python27\python.exe runserver.py -a ptc -u ***** -p ***** -l "1-41-6 Takashimadaira Itabashi, Tokyo, Japan" -st 10
2016-07-21 21:21:14,938 [  runserver] [   INFO] Parsed location is: 35.7840/139.6666/0.0000 (lat/lng/alt)
2016-07-21 21:21:14,941 [     search] [   INFO] Attempting login to Pokemon Go.
2016-07-21 21:21:14,950 [   auth_ptc] [   INFO] PTC login for: *****
2016-07-21 21:21:14,954 [  _internal] [   INFO]  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2016-07-21 21:21:15,437 [   auth_ptc] [   INFO] PTC Login successful
2016-07-21 21:21:15,937 [     search] [   INFO] Login to Pokemon Go successful.
2016-07-21 21:21:15,938 [     search] [   INFO] Scanning step 1 of 100.
2016-07-21 21:21:16,328 [     search] [  ERROR] Scan step failed. Response dictionary key error.
Exception in thread search_thread:
Traceback (most recent call last):
  File "C:\Python27\lib\threading.py", line 801, in __bootstrap_inner
    self.run()
  File "C:\Python27\lib\threading.py", line 754, in run
    self.__target(*self.__args, **self.__kwargs)
  File "C:\Users\*****\Documents\PokemonGoMap\PokemonGo-Map-develop\pogom\search.py", line 99, in search_loop
    search(args)
  File "C:\Users\*****\Documents\PokemonGoMap\PokemonGo-Map-develop\pogom\search.py", line 87, in search
    failed_consecutive += 1
UnboundLocalError: local variable 'failed_consecutive' referenced before assignment

* Version used: Version as of today.
* Environment name and version (e.g. Python 2.7): 2.7.12
* Operating System and version (desktop or mobile): Windows 10

**Comment by takutok** [comment]
"1 Chome-28-5 Takashimadaira, Itabashi, Tokyo, Japan"

I was able in this.

**Comment by takutok** [comment]
"1 Chome-41-6 Takashimadaira, Itabashi, Tokyo, Japan"

I was able in this.

**Comment by macgngsta** [comment]
this might be fixed already in dfe9b88: 

````
 global failed_consecutive
            failed_consecutive += 1
````
